### DISCUSSION TITLE
How to use `description` to compile hotkeys in current scope?

### DISCUSSION THREAD

**Discussion by sirvine** [JohannesKlauss/react-hotkeys-hook#971] [General]
I notice that the README indicates that the `description` option is "helpful if you want to display a list of active hotkeys to the user."

My hope was that descriptions would appear in the `hotkeys` objects returned by:

```
const { hotkeys } = useHotkeysContext();
```

It looks like they're not, so I wonder if there is another way to compile a list of currently-in-scope hotkeys that includes the values for `description`?

**Comment by JohannesKlauss** [comment]
You are right this feature doesn't exist yet. Could you open an issue as a Feature Request? Thank you.

**Reply by sirvine** [reply]
Will do. Thanks!

**Reply by zenzen-sol** [reply]
Added feature request #973 

I also did PR #972 for inspiration, but feel free to close if you prefer another implementation.

**Reply by zenzen-sol** [reply]
(sorry about bouncing between two github accounts).
